webpack.common.js

```diff
@@ -82,16 +82,15 @@ module.exports = {
     extensions: [".ts", ".tsx", ".js", ".png", ".svg"],
     alias: {
       "@src": path.resolve(__dirname, "src/"),
+      // snarkjs uses ejs which has unsafe-eval function constructor
+      ejs: path.resolve(__dirname, "src/config/ejsMock.js"),
       buffer: "buffer",
     },
     fallback: {
-      browserify: require.resolve("browserify"),
```

Comment:
Do we still need this package in the `package.json`? 
https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/60713467b351f728894048f83a4803d813131a42/package.json#L55

Reply:
This field is not required here. It works well without it.